conans/test/integration/command_v2/test_outdated.py

```diff
@@ -0,0 +1,194 @@
+import json
+from collections import OrderedDict
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient, TestServer
+
+
+def test_outdated_command():
+    tc = TestClient(default_server_user=True)
+    #Create libraries needed to generate the dependency graph
```

Comment:
```suggestion
    # Create libraries needed to generate the dependency graph
```

If you activate a linter, this will complain